h/browser/chrome/lib/uri-info.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * A wrapper for the server's uriinfo endpoint.
+ *
+ * Exports a function that makes an HTTP request to the server's uriinfo
+ * endpoint and returns a Promise that resolves to the response. Usage:
+ *
+ *     var uriInfo = require('uri-info');
```

Comment:
I'm not sure what you want me to change here? This is consistent with other modules, eg. `sidebar-injector.js` is imported as `SidebarInjector`.

I've corrected some other places where `uri-info.js` was imported as `uriinfo` (lowercase) though